packages/block-library/src/gallery/editor.scss

```diff
@@ -206,3 +206,15 @@ figure.wp-block-gallery {
 	// Some themes give all <ul> default margin instead of padding.
 	margin: 0;
 }
+
+.wp-block-update-gallery-modal {
+	max-width: 400px;
+	.wp-block-update-gallery-modal-buttons {
+		display: flex;
+		justify-content: flex-end;
+	
```

Comment:
Tiniest of nits: the linting step is failing because of a single whitespace character on this line 😄

![image](https://user-images.githubusercontent.com/14988353/132436400-df02cdb1-1b0c-4a34-9273-4edb8521bf23.png)